docs/source/tutorials/e2e_flow.rst

```diff
@@ -522,5 +522,31 @@ The output should look something like this:
 
 And thats it! Try your own prompt!
 
+Uploading your model to the Hugging Face Hub
+--------------------------------------------
+
+Your new model is working great and you want to share it with the world. The easiest way to do this
+is utilizing the ``huggingface-cli`` command, which is integrated with torchtune. Simply point the CLI
```

Comment:
nit: I don't believe it's integrated anymore, its more like interops or works seamlessly with